config/main.py

```diff
@@ -935,8 +899,7 @@ def config(ctx):
 
     ctx.obj = Db()
 
-pass_db = click.make_pass_decorator(Db, ensure=True)
-
+config.add_command(feature.feature)
```

Comment:
Suggest to move this after the `aaa` `add_command()` lines to keep them in alphabetical order, and add a newline after.